### ISSUE TITLE
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)

### ISSUE THREAD

**Issue by Pilly170** [decoxviii/userrecon-py#1]
Ran the tool, along with the results got this error with each account
```
SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f856c4c5710>
transport: <_SelectorSocketTransport fd=7 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)
```

**Comment by decoxviii** [comment]
Hi @Pilly170,

Please could you show me the output of the following commands:
```
python3 --version
```
and
```
pip3 freeze | grep aiohttp
```

**Comment by Lifedealer** [comment]
I get the same error.

`python3 --version`
Python 3.7.3

`pip3 freeze | grep aiohttp`
aiohttp==3.5.4

**Comment by decoxviii** [comment]
This problem is related to python and is expected to be fixed in version `3.7.4` If you can not wait for this version to be released, you can run the tool with python `3.6.7`:

```
git clone https://github.com/decoxviii/userrecon-py.git ; cd userrecon-py
python3.6 bin/userrecon-py target 'username'
```

Reference: [aio-libs/aiohttp#3535](https://github.com/aio-libs/aiohttp/issues/3535)
